tests/app/shared/comment-editor/comment-editor.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { DebugElement, ElementRef } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormControl, FormGroup } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+import { MarkdownModule } from 'ngx-markdown';
+
+import { ErrorHandlingService } from '../../../../src/app/core/services/error-handling.service';
+import { LoggingService } from '../../../../src/app/core/services/logging.service';
+import { UploadService } from '../../../../src/app/core/services/upload.service';
+import { CommentEditorComponent } from '../../../../src/app/shared/comment-editor/comment-editor.component';
+import { DISPLAYABLE_CONTENT, insertUploadingText, insertUploadUrl } from '../../../../src/app/shared/comment-editor/upload-text-insertor';
+import { SharedModule } from '../../../../src/app/shared/shared.module';
+
+function isDisplayable(filename) {
+  return DISPLAYABLE_CONTENT.includes(filename.split('.').pop().toLowerCase());
+}
+
+describe('CommentEditorComponent', () => {
```

Comment:
Since this file is not for testing ```CommentEditorComponent``` but the methods within ```upload-text-insertor``` file, you should not start the describe block with CommentEditorComponent.

Reply:
Ok, this change has been made 👍